lib/modules/dosomething/dosomething_reportback/dosomething_reportback.admin.inc

```diff
@@ -282,9 +284,30 @@ function dosomething_reportback_files_form_submit($form, &$form_state) {
       // Flush the image style cache for this image so they update with the new image.
       image_path_flush($image->source);
     }
+
+    // Check to see if Rogue feature flag is enabled.
+    if (module_exists('dosomething_rogue')) {
+      $rogue_item_id = dosomething_rogue_get_by_file_id($fid);
+
+      // If there is a rogue_item_id in the dosomething_rogue_reportbacks table, send update to Rogue.
+      if (!is_null($rogue_item_id[0]->rogue_item_id)) {
+        $data = [
+          'rogue_reportback_item_id' => $rogue_item_id[0]->rogue_item_id,
+          'status' => $values['status'],
+        ];
+
+        array_push($updates_to_send_to_rogue, $data);
+      }
+    }
   }
 
-  drupal_set_message(t("Updated."));
+  // Send updates to Rogue.
+  $response = dosomething_rogue_update_rogue_reportback_items($updates_to_send_to_rogue);
+  if ($response->code != 201) {
+    drupal_set_message(print_r($response->code, TRUE) . ' ' . print_r($response->status_message, TRUE), 'warning');
```

Comment:
@sbsmith86 @angaither got this to work finally and made it more broad to not only show error but any status code and message if not a `201`. If we get this message, it basically means that everything is updated in Phoenix but request to Rogue didn't successfully go through - should I add more context to this in the message besides the code and message?